Surface_sweep_2/include/CGAL/Surface_sweep_2/Random_access_output_iterator.h

```diff
@@ -0,0 +1,105 @@
+// Copyright (c) 2006,2007,2009,2010,2011 Tel-Aviv University (Israel).
```

Comment:
Maybe you should use "git mv"?

Reply:
I did, Github interprets this as deleted file + new file, I can't really do much about it.